aggregator-options/grix/index.ts

```diff
@@ -0,0 +1,47 @@
+import { SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { httpGet } from "../../utils/fetchURL";
+import { GrixMetricsData } from "../../adapters/types";
+
+const fetchGrix = async () => {
+  const url = `https://internal-api-dev.grix.finance/grixmetrics`;
+
+  try {
+    const grixMetricsResponse = await httpGet(url);
+    const grixMetricsData = parseGrixMetricsData(grixMetricsResponse);
+
+    const stats = grixMetricsData ? extractStats(grixMetricsData) : null;
+
+    return { totalNotionalVolume: stats?.totalNotionalVolume };
+  } catch (e: any) {
+    throw e;
+  }
+};
+
+const parseGrixMetricsData = (result: any): GrixMetricsData | null => {
+  if (typeof result === "object" && result !== null) {
+    return result as GrixMetricsData;
+  }
+  return result ? (JSON.parse(result) as GrixMetricsData) : null;
+};
+
+const extractStats = (data: GrixMetricsData) => ({
+  totalNotionalVolume: Number(data.graphStatistics.totalNotionalValue),
+});
+
+const grix_adapter: SimpleAdapter = {
+  version: 1,
+  adapter: {
+    [CHAIN.ARBITRUM]: {
+      fetch: fetchGrix,
+      start: "2024-11-01",
+      runAtCurrTime: false, // currently we don't take the timestamp into account, should be changed soon
```

Comment:
then, this field should be set as true as the returned result is always for current time